### ISSUE TITLE
Enhance Footer

### ISSUE THREAD

**Issue by foohonpie** [pbbg/pbbg.com#31]
It'd be nice to more directly acknowledge our active community members.  Let's change:

> by Foohon Pie LLC with help from [friends and supporters](https://pbbg.com/humans.txt)

to

> by Foohon Pie LLC with help from [friends](https://discord.gg/KvwaEqC), [supporters](https://www.patreon.com/foohonpie), and [contributors](https://github.com/pbbg).

**Comment by krzysztof-sikorski** [comment]
Does that mean the `humans.txt` file will be removed, or just hidden as a kind of easter egg?

**Comment by foohonpie** [comment]
Easter Egg sounds nice.  We'll need to update it at some point, and automate the list of Patreon subs and GitHub contributors.

**Comment by bpkennedy** [comment]
Additional Acceptance Criteria for this story:
1) display the package.json version number (ie v1.0.1) as a hyperlink in the footer.
2) clicking said hyperlink routes to a simple changelog view that displays the content of a .md file.

We need not add entries to the markdown file for every change - we can decide to do that separately from the actual version.  For example maybe we only add to the changelog.md when significant features are added.

The format of the changelog can be like:
```
## [1.0.1] - 2020-05-19
 - Added game news feed to home page
 - Fixed feedback to prevent multiple reviews on a game by the same user.
```

The version number can double as a troubleshooting aid for us - we use it to verify if the latest code is pushed to that environment or not.

**Comment by foohonpie** [comment]
Closed by #34.
